configuration/arm64/initialization.sh

```diff
@@ -11,3 +38,12 @@ sudo apt-get -y install docker-ce docker-ce-cli containerd.io docker-compose-plu
 sudo apt -y install python3-pip
 sudo pip3 install docker-compose
 sudo docker-compose up -d
+
+#create v2xhub user
+cd mysql
+./add_v2xhub_user.bash
+
+sudo apt-get update -y && sudo apt update -y
```

Comment:
Why are we calling a second update here?

Reply:
will remove